docs/utilization.md

```diff
@@ -0,0 +1,116 @@
+# Utilization reports in Tock
+
+[:arrow_left: Back to Tock Documentation](../docs)
+
+
+Tock utilization reports display progress towards billing targets
+for each user, business unit, and the overall organization.
+
+Each user has a billing target which is the percentage of worked hours that are expected to be tocked to billable projects.
+
+We calculate utilization, or progress, towards these targets by totaling the hours worked on billable projects and dividing by the total number of hours which were worked.
+
+A few example calculations are included below.
+
+## Definitions
+
+### Utilization
+The percentage of `BILLABLE HOURS` to `TARGET-BILLABLE-HOURS` tocked over a period of time.
+
+    < 100% - a user has not reached their billable goal
+
+    100% - a user has reached their billable goal
+
+    > 100% - a user has exceeded their billable goal
+
+### BILLABLE HOURS
+The sum of hours tocked to billable projects.
+
+### NON BILLABLE HOURS
+The sum of hours tocked to non-billable projects, excluding hours which were not worked like `Out of Office`.
+
+### TARGET BILLABLE HOURS
+A user's expected number of `BILLABLE HOURS` over a period of time. Determined by their `BILLING EXPECTATION` over that period and the total number of billable and non-billable hours tocked.
+
+```python
+TARGET_BILLABLE_HOURS = (BILLABLE_HOURS + NON_BILLABLE_HOURS) * BILLING_EXPECTATION
+```
+
+### BILLING EXPECTATION
+
+The percent of `TOTAL HOURS` which are expected to be billable for a given reporting period. Set per user, per timecard, and will vary as an individual's role, position, or organization change.
+
+**Default: 80%**
+
+## Example
+
+Stacey is an active and billable 18F employee, here's a look at their recent tock activity! As an independent contributor, Stacey has a `BILLING_EXPECTATION` of 80%.
```

Comment:
Minor, but I think you may have meant "individual contributor"

Reply:
Ah yes!